src/Megamix/Patches.cpp

```diff
@@ -0,0 +1,125 @@
+#include <array>
+
+#include <3ds.h>
+#include <CTRPluginFramework.hpp>
+
+#include "Megamix.hpp"
+#include "Megamix/Patches.hpp"
+
+namespace Megamix::Patches {
+    const u32 MuseumRowCount = 32;
+
+    const MuseumRow MuseumRows[MuseumRowCount] {
+        /* E2 */ MuseumRow({ 0x049, 0x04a, 0x02c, 0x101, 0x101 }, (char *) 0x50201b, 0, 0),
+        /* E1 */ MuseumRow({ 0x069, 0x068, 0x06b, 0x06a, 0x101 }, (char *) 0x50201b, 0, 0),
+        /* E0 */ MuseumRow({ 0x023, 0x05b, 0x006, 0x00e, 0x05d }, (char *) 0x50201b, 0, 0),
+
+        /* 0  */ MuseumRow({ 0x059, 0x005, 0x007, 0x00d, 0x101 }, (char *) 0x50215a, 0, 0),
+        /* 1  */ MuseumRow({ 0x002, 0x003, 0x00a, 0x00b, 0x101 }, (char *) 0x50217e, 0, 1),
+        /* 2  */ MuseumRow({ 0x069, 0x101, 0x101, 0x101, 0x101 }, (char *) 0x502141, 0, 2),
+        /* 3  */ MuseumRow({ 0x000, 0x006, 0x008, 0x00c, 0x101 }, (char *) 0x5021a2, 0, 3),
+        /* 4  */ MuseumRow({ 0x017, 0x01d, 0x02d, 0x044, 0x101 }, (char *) 0x5021bb, 0, 4),
+        /* 5  */ MuseumRow({ 0x068, 0x101, 0x101, 0x101, 0x101 }, (char *) 0x502165, 0, 5),
+        /* 6  */ MuseumRow({ 0x001, 0x004, 0x009, 0x00e, 0x101 }, (char *) 0x5021c6, 0, 6),
```

Comment:
it might be a good idea to make the indexes an enum (like `Slot::AgbBatterL` or something), but i'm not sure if that'll damage legibility. thoughts?